boards/pba-d-01-kw2x/doc.txt

```diff
@@ -9,29 +9,54 @@ Designed and produced by PHYTEC Messtechnik GmbH, D-55129 Mainz.
 contact@phytec.de
 
 ## Overview
-The [Phytec IoT wiki](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki)
-describes the phyWAVE module and the phyNODE board more
-detailed. The links below will guide you directly to the corresponding chapter:
- * [Overview](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/)
- * [Introduction](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/Introduction)
- * Hardware descriptions for the
-[phyWAVE](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/phyWAVE-KW2x-Characteristics)
-and the
-[phyNODE](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/phyNODE-Characteristics)
- * [Toolchain, build and debug Information](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/RIOT-getting-started#toolchain-summary)
- * [A detailed step-by-step Guide that explains how to set up the Environment](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/RIOT-getting-started#step-by-step-installation-guide)
+
+The phyNODE evaluation board consists of a [phyWAVE](https://www.phytec.de/produkte/internet-of-things/phywave/)
+module mounted on a common PCB with several sensor and actuator modules.
+The board provides 2 USB ports, one for development (debug and flash) and the
+other to be used by the firmware.
+
 The implementation status of the phyWAVE and phyNODE peripherals are listed
 below. You can find links to the corresponding Pull Requests which may give you
 a code reference.
 
 <img src="https://raw.githubusercontent.com/jremmert-phytec-iot/Pictures_Phytec_IOT/master/EvalBoard_3.png" width="800" />
 phyNODE board with equipped phyWAVE-KW2x processor/radio-module.
 
-### Details
-The PCB-pieces with the mounted sensors can be separated from the evaluation
-board to capture sensor values from specific spatial points.
+@note The PCB-pieces with the mounted sensors can be separated from the
+      evaluation board to capture sensor values from specific spatial points.
+
+## Pinout Reference
+
+The board features an Arduino like header connector to mount shields.The spacing
+of the 4 pin rows is a bit offset, thus shields might not fit perfectly.
+
+@note Not all signals and pins are compatible with Arduino Uno R3. Also check
+      the pin configuration in RIOT, i.e., see `periph_conf.h` of the board.
 
-[Pinout reference](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/Interfaces)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+                             _X1____________________
+                            |                   D15 |___DIO21, PE19, I2C0 SCL
```

Comment:
while the name I2C0 refers to the manufacturer description it might be nice to also (or instead) provide information on how it is actually configured by default in RIOT. The DIOxy pin names on the other hand are probably not interesting to most (any?) RIOT users.

Reply:
the idea is to show the (complete) vendor provided naming of pins as a reference, independent of the (current) RIOT config of the board.

Reply:
I'm fine with that too, but then the above "Also check the pin configuration in RIOT, i.e., see 'periph_conf.h' of the board." should be more explicitly telling the user that this is referring *only* to the manufacturer/MCU names and *not* referring to the periph instance identifiers of RIOT. Btw. a direct link to the periph_conf would  also help :)

Reply:
How about this one?

Reply:
done